<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8"/>
    <title>Test Report</title>
    <link href="assets/style.css" rel="stylesheet" type="text/css"/></head>
  <body onLoad="init()">
    <script>/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */


function toArray(iter) {
    if (iter === null) {
        return null;
    }
    return Array.prototype.slice.call(iter);
}

function find(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return elem.querySelector(selector);
}

function find_all(selector, elem) {
    if (!elem) {
        elem = document;
    }
    return toArray(elem.querySelectorAll(selector));
}

function sort_column(elem) {
    toggle_sort_states(elem);
    var colIndex = toArray(elem.parentNode.childNodes).indexOf(elem);
    var key;
    if (elem.classList.contains('numeric')) {
        key = key_num;
    } else if (elem.classList.contains('result')) {
        key = key_result;
    } else {
        key = key_alpha;
    }
    sort_table(elem, key(colIndex));
}

function show_all_extras() {
    find_all('.col-result').forEach(show_extras);
}

function hide_all_extras() {
    find_all('.col-result').forEach(hide_extras);
}

function show_extras(colresult_elem) {
    var extras = colresult_elem.parentNode.nextElementSibling;
    var expandcollapse = colresult_elem.firstElementChild;
    extras.classList.remove("collapsed");
    expandcollapse.classList.remove("expander");
    expandcollapse.classList.add("collapser");
}

function hide_extras(colresult_elem) {
    var extras = colresult_elem.parentNode.nextElementSibling;
    var expandcollapse = colresult_elem.firstElementChild;
    extras.classList.add("collapsed");
    expandcollapse.classList.remove("collapser");
    expandcollapse.classList.add("expander");
}

function show_filters() {
    var filter_items = document.getElementsByClassName('filter');
    for (var i = 0; i < filter_items.length; i++)
        filter_items[i].hidden = false;
}

function add_collapse() {
    // Add links for show/hide all
    var resulttable = find('table#results-table');
    var showhideall = document.createElement("p");
    showhideall.innerHTML = '<a href="javascript:show_all_extras()">Show all details</a> / ' +
                            '<a href="javascript:hide_all_extras()">Hide all details</a>';
    resulttable.parentElement.insertBefore(showhideall, resulttable);

    // Add show/hide link to each result
    find_all('.col-result').forEach(function(elem) {
        var collapsed = get_query_parameter('collapsed') || 'Passed';
        var extras = elem.parentNode.nextElementSibling;
        var expandcollapse = document.createElement("span");
        if (collapsed.includes(elem.innerHTML)) {
            extras.classList.add("collapsed");
            expandcollapse.classList.add("expander");
        } else {
            expandcollapse.classList.add("collapser");
        }
        elem.appendChild(expandcollapse);

        elem.addEventListener("click", function(event) {
            if (event.currentTarget.parentNode.nextElementSibling.classList.contains("collapsed")) {
                show_extras(event.currentTarget);
            } else {
                hide_extras(event.currentTarget);
            }
        });
    })
}

function get_query_parameter(name) {
    var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
    return match && decodeURIComponent(match[1].replace(/\+/g, ' '));
}

function init () {
    reset_sort_headers();

    add_collapse();

    show_filters();

    sort_column(find('.initial-sort'));

    find_all('.sortable').forEach(function(elem) {
        elem.addEventListener("click",
                              function(event) {
                                  sort_column(elem);
                              }, false)
    });

};

function sort_table(clicked, key_func) {
    var rows = find_all('.results-table-row');
    var reversed = !clicked.classList.contains('asc');
    var sorted_rows = sort(rows, key_func, reversed);
    /* Whole table is removed here because browsers acts much slower
     * when appending existing elements.
     */
    var thead = document.getElementById("results-table-head");
    document.getElementById('results-table').remove();
    var parent = document.createElement("table");
    parent.id = "results-table";
    parent.appendChild(thead);
    sorted_rows.forEach(function(elem) {
        parent.appendChild(elem);
    });
    document.getElementsByTagName("BODY")[0].appendChild(parent);
}

function sort(items, key_func, reversed) {
    var sort_array = items.map(function(item, i) {
        return [key_func(item), i];
    });

    sort_array.sort(function(a, b) {
        var key_a = a[0];
        var key_b = b[0];

        if (key_a == key_b) return 0;

        if (reversed) {
            return (key_a < key_b ? 1 : -1);
        } else {
            return (key_a > key_b ? 1 : -1);
        }
    });

    return sort_array.map(function(item) {
        var index = item[1];
        return items[index];
    });
}

function key_alpha(col_index) {
    return function(elem) {
        return elem.childNodes[1].childNodes[col_index].firstChild.data.toLowerCase();
    };
}

function key_num(col_index) {
    return function(elem) {
        return parseFloat(elem.childNodes[1].childNodes[col_index].firstChild.data);
    };
}

function key_result(col_index) {
    return function(elem) {
        var strings = ['Error', 'Failed', 'Rerun', 'XFailed', 'XPassed',
                       'Skipped', 'Passed'];
        return strings.indexOf(elem.childNodes[1].childNodes[col_index].firstChild.data);
    };
}

function reset_sort_headers() {
    find_all('.sort-icon').forEach(function(elem) {
        elem.parentNode.removeChild(elem);
    });
    find_all('.sortable').forEach(function(elem) {
        var icon = document.createElement("div");
        icon.className = "sort-icon";
        icon.textContent = "vvv";
        elem.insertBefore(icon, elem.firstChild);
        elem.classList.remove("desc", "active");
        elem.classList.add("asc", "inactive");
    });
}

function toggle_sort_states(elem) {
    //if active, toggle between asc and desc
    if (elem.classList.contains('active')) {
        elem.classList.toggle('asc');
        elem.classList.toggle('desc');
    }

    //if inactive, reset all other functions and add ascending active
    if (elem.classList.contains('inactive')) {
        reset_sort_headers();
        elem.classList.remove('inactive');
        elem.classList.add('active');
    }
}

function is_all_rows_hidden(value) {
  return value.hidden == false;
}

function filter_table(elem) {
    var outcome_att = "data-test-result";
    var outcome = elem.getAttribute(outcome_att);
    class_outcome = outcome + " results-table-row";
    var outcome_rows = document.getElementsByClassName(class_outcome);

    for(var i = 0; i < outcome_rows.length; i++){
        outcome_rows[i].hidden = !elem.checked;
    }

    var rows = find_all('.results-table-row').filter(is_all_rows_hidden);
    var all_rows_hidden = rows.length == 0 ? true : false;
    var not_found_message = document.getElementById("not-found-message");
    not_found_message.hidden = !all_rows_hidden;
}
</script>
    <h1>bg.html</h1>
    <p>Report generated on 07-Nov-2020 at 23:00:57 by <a href="https://pypi.python.org/pypi/pytest-html">pytest-html</a> v2.0.1</p>
    <h2>Environment</h2>
    <table id="environment">
      <tr>
        <td>JAVA_HOME</td>
        <td>C:\Program Files\Java\jdk1.8.0_31</td></tr>
      <tr>
        <td>Packages</td>
        <td>{&apos;pytest&apos;: &apos;5.3.5&apos;, &apos;py&apos;: &apos;1.8.0&apos;, &apos;pluggy&apos;: &apos;0.13.0&apos;}</td></tr>
      <tr>
        <td>Platform</td>
        <td>Windows-10-10.0.17134-SP0</td></tr>
      <tr>
        <td>Plugins</td>
        <td>{&apos;allure-pytest&apos;: &apos;2.8.6&apos;, &apos;celery&apos;: &apos;4.3.0&apos;, &apos;html&apos;: &apos;2.0.1&apos;, &apos;metadata&apos;: &apos;1.8.0&apos;}</td></tr>
      <tr>
        <td>Python</td>
        <td>3.7.0</td></tr></table>
    <h2>Summary</h2>
    <p>1 tests ran in 13.17 seconds. </p>
    <p class="filter" hidden="true">(Un)check the boxes to filter the results.</p><input checked="true" class="filter" data-test-result="passed" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="passed">1 passed</span>, <input checked="true" class="filter" data-test-result="skipped" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="skipped">0 skipped</span>, <input checked="true" class="filter" data-test-result="failed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="failed">0 failed</span>, <input checked="true" class="filter" data-test-result="error" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="error">0 errors</span>, <input checked="true" class="filter" data-test-result="xfailed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xfailed">0 expected failures</span>, <input checked="true" class="filter" data-test-result="xpassed" disabled="true" hidden="true" name="filter_checkbox" onChange="filter_table(this)" type="checkbox"/><span class="xpassed">0 unexpected passes</span>
    <h2>Results</h2>
    <table id="results-table">
      <thead id="results-table-head">
        <tr>
          <th class="sortable result initial-sort" col="result">Result</th>
          <th class="sortable" col="name">Test</th>
          <th class="sortable numeric" col="duration">Duration</th>
          <th>Links</th></tr>
        <tr hidden="true" id="not-found-message">
          <th colspan="4">No results found. Try to check the filters</th></tr></thead>
      <tbody class="passed results-table-row">
        <tr>
          <td class="col-result">Passed</td>
          <td class="col-name">test_api.py::TestRun::test_go_on_run</td>
          <td class="col-duration">12.86</td>
          <td class="col-links"></td></tr>
        <tr>
          <td class="extra" colspan="4">
            <div class="log"> -------------------------------Captured log call-------------------------------- <br/>ERROR    root:print_log.py:38 HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96206A908&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))
Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 160, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File &quot;D:\软件\lib\site-packages\urllib3\util\connection.py&quot;, line 80, in create_connection
    raise err
  File &quot;D:\软件\lib\site-packages\urllib3\util\connection.py&quot;, line 70, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 603, in urlopen
    chunked=chunked)
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 355, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1016, in _send_output
    self.send(msg)
  File &quot;D:\软件\lib\http\client.py&quot;, line 956, in send
    self.connect()
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 183, in connect
    conn = self._new_conn()
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 169, in _new_conn
    self, &quot;Failed to establish a new connection: %s&quot; % e)
urllib3.exceptions.NewConnectionError: &lt;urllib3.connection.HTTPConnection object at 0x000002D96206A908&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\requests\adapters.py&quot;, line 449, in send
    timeout=timeout
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 641, in urlopen
    _stacktrace=sys.exc_info()[2])
  File &quot;D:\软件\lib\site-packages\urllib3\util\retry.py&quot;, line 399, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96206A908&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\autoapi\python3_interface-master\main\test_api.py&quot;, line 91, in test_go_on_run
    res = RunMethod().run_main(method,url,data,header,params=data)
  File &quot;D:\autoapi\python3_interface-master\util\runmethod.py&quot;, line 25, in run_main
    res = self.post_main(url,data,header)
  File &quot;D:\autoapi\python3_interface-master\util\runmethod.py&quot;, line 9, in post_main
    res = requests.post(url=url,data=data,headers=header)
  File &quot;D:\软件\lib\site-packages\requests\api.py&quot;, line 116, in post
    return request(&#x27;post&#x27;, url, data=data, json=json, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\api.py&quot;, line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\sessions.py&quot;, line 533, in request
    resp = self.send(prep, **send_kwargs)
  File &quot;D:\软件\lib\site-packages\requests\sessions.py&quot;, line 646, in send
    r = adapter.send(request, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\adapters.py&quot;, line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96206A908&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))
ERROR    root:print_log.py:38 list index out of range
Traceback (most recent call last):
  File &quot;D:\autoapi\python3_interface-master\main\test_api.py&quot;, line 70, in test_go_on_run
    depend_response_data = self.depend_data.get_value_for_key(i)
  File &quot;D:\autoapi\python3_interface-master\data\dependent_data.py&quot;, line 49, in get_value_for_key
    newdepend+=str([match.value for match in parse(item).find(response_data)][0])+&quot;,&quot;
IndexError: list index out of range
ERROR    root:print_log.py:38 HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96227E278&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))
Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 160, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw)
  File &quot;D:\软件\lib\site-packages\urllib3\util\connection.py&quot;, line 80, in create_connection
    raise err
  File &quot;D:\软件\lib\site-packages\urllib3\util\connection.py&quot;, line 70, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 603, in urlopen
    chunked=chunked)
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 355, in _make_request
    conn.request(method, url, **httplib_request_kw)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1229, in request
    self._send_request(method, url, body, headers, encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1275, in _send_request
    self.endheaders(body, encode_chunked=encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1224, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File &quot;D:\软件\lib\http\client.py&quot;, line 1016, in _send_output
    self.send(msg)
  File &quot;D:\软件\lib\http\client.py&quot;, line 956, in send
    self.connect()
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 183, in connect
    conn = self._new_conn()
  File &quot;D:\软件\lib\site-packages\urllib3\connection.py&quot;, line 169, in _new_conn
    self, &quot;Failed to establish a new connection: %s&quot; % e)
urllib3.exceptions.NewConnectionError: &lt;urllib3.connection.HTTPConnection object at 0x000002D96227E278&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\软件\lib\site-packages\requests\adapters.py&quot;, line 449, in send
    timeout=timeout
  File &quot;D:\软件\lib\site-packages\urllib3\connectionpool.py&quot;, line 641, in urlopen
    _stacktrace=sys.exc_info()[2])
  File &quot;D:\软件\lib\site-packages\urllib3\util\retry.py&quot;, line 399, in increment
    raise MaxRetryError(_pool, url, error or ResponseError(cause))
urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96227E278&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File &quot;D:\autoapi\python3_interface-master\main\test_api.py&quot;, line 91, in test_go_on_run
    res = RunMethod().run_main(method,url,data,header,params=data)
  File &quot;D:\autoapi\python3_interface-master\util\runmethod.py&quot;, line 25, in run_main
    res = self.post_main(url,data,header)
  File &quot;D:\autoapi\python3_interface-master\util\runmethod.py&quot;, line 9, in post_main
    res = requests.post(url=url,data=data,headers=header)
  File &quot;D:\软件\lib\site-packages\requests\api.py&quot;, line 116, in post
    return request(&#x27;post&#x27;, url, data=data, json=json, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\api.py&quot;, line 60, in request
    return session.request(method=method, url=url, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\sessions.py&quot;, line 533, in request
    resp = self.send(prep, **send_kwargs)
  File &quot;D:\软件\lib\site-packages\requests\sessions.py&quot;, line 646, in send
    r = adapter.send(request, **kwargs)
  File &quot;D:\软件\lib\site-packages\requests\adapters.py&quot;, line 516, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPConnectionPool(host=&#x27;localhost&#x27;, port=6777): Max retries exceeded with url: /xinxin/user/register (Caused by NewConnectionError(&#x27;&lt;urllib3.connection.HTTPConnection object at 0x000002D96227E278&gt;: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。&#x27;))<br/></div></td></tr></tbody></table></body></html>